data/ships.txt

```diff
@@ -171,30 +171,30 @@ ship "Bactrian"
 	attributes
 		category "Heavy Warship"
 		"cost" 17600000
-		"shields" 17500
-		"hull" 8600
+		"shields" 12400
+		"hull" 9800
 		"required crew" 70
```

Comment:
Worth mentioning that the Bactrian--a clunky civilian city-ship from an older time--has a lower stock crew than the Hurricane--a tier 2 warship designed by a species obsessed with efficiency.